src/components/List.js

```diff
@@ -1,9 +1,21 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
 
-const List = ({ token, results }) => {
+const List = ({ results, setSearchTerm, searchTerm }) => {
   return (
     <>
+      <form>
+        <label htmlFor="searchField">Search</label>
+        <br />
```

Comment:
Should we use CSS rather than inline styling to put these elements on separate lines? `<br>` seems to have some accessibility concerns.  https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br